packages/fuels-test-helpers/src/lib.rs

```diff
@@ -190,7 +193,9 @@ pub async fn setup_test_client(
     } else if is_free(requested_port) {
         config.addr
     } else {
-        panic!("Error: Address already in use");
+        return Err(fuels_core::types::errors::Error::IOError(
+            std::io::ErrorKind::AddrInUse.into(),
+        ));
```

Comment:
If you import `Error` you can shorten this to:
```suggestion
        return Err(Error::IOError(std::io::ErrorKind::AddrInUse.into()));
```

Reply:
had to add allow(unused_imports) to to silence the cargo check and clippy